scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -232,36 +235,149 @@ def release_paths(self) -> List[Path]:
         return release_paths
 
 
+class ModelType(Enum):
+    """Enum for the Model Types accepted in ModelConfig."""
+
+    MLFLOW = "mlflow_model"
+    CUSTOM = "custom_model"
+    TRITON = "triton_model"

```

Comment:
>triton_model [](http://example.com/codeflow?start=14&length=12)

@wayliums @elesel Could you please provide more details on what Triton model is?